src/main/java/edu/ucsb/courses/documents/statistics/SingleCourseSearch.java

```diff
@@ -0,0 +1,78 @@
+package edu.ucsb.courses.documents.statistics;
+
+import java.util.List;
```

Comment:
This is an unused import, we should remove.

Reply:
We could not remove the package since it would not run without it.

Reply:
We removed the unused import